spec/did_metadata.md

```diff
@@ -88,9 +88,13 @@ to the DID that has been resolved. It is similar to the `alsoKnownAs` DID docume
 [Also Known As](#also-known-as)), but it has even stronger semantics, insofar as the logical equivalence is
 guaranteed by the DID method itself.
 
-In the case of `did:webs`, this metadata property SHOULD contain a list of other known `did:webs` DIDs that differ
+In the case of `did:webs`, this metadata property SHOULD contain a list of the controller AID [[ref: designated aliases]] `did:webs` DIDs that differ
 in the domain name and/or port portion of the [[ref: method-specific identifier]]
-but share the same AID. Also see section [Equivalent Identifiers](#equivalent-identifiers).
+but share the same AID. Also see section [[ref:AID controlled identifiers]].
+
+Note that [[ref:AID controlled identifiers]] like `did:web` and `did:keri` identifiers with the same AID are not listed in `equivalentId` because they do not have the same DID method. A `did:web` identifier with the same domain and AID does not have the same security characteristics as the `did:webs` identifier. Conversly, a `did:keri` identifier with the same AID has the same security characterisitcs but not the same dependence on the web. For these reasons, they are not listed in `equivalentId`.
```

Comment:
```suggestion
Note that [[ref:AID controlled identifiers]] like `did:web` and `did:keri` identifiers with the same AID are not listed in `equivalentId` because they do not have the same DID method. A `did:web` identifier with the same domain and AID does not have the same security characteristics as the `did:webs` identifier. Conversely, a `did:keri` identifier with the same AID has the same security characterisitcs but not the same dependence on the web. For these reasons, they are not listed in `equivalentId`.
```